### PR TITLE
Add code snippets for firestore modules.

### PR DESCRIPTION
**Pull request by jkyle109**
Added code snippets to support documentation for Firestore modules.

Related to https://github.com/firebase/firebase-admin-python/issues/104

### CODE REVIEW

File: snippets/firestore/firestore.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+    # [START add_data]
+    doc_ref = db.collection("users").document("alovelace")
+    doc_ref.set({
+        "first": "Ada",
+        "last": "Lovelace",
+        "born": 1815
+    })
+    # [END add_data]
+
```
Comment (lahirumaramba) [inline snippets/firestore/firestore.py:85]: nit: new line

File: snippets/firestore/firestore_async.py [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import asyncio
+
+from firebase_admin import firestore_async
+
+# pylint: disable=invalid-name
+def init_firestore_client():
+    # [START init_firestore_client]
```
Comment (lahirumaramba) [inline snippets/firestore/firestore_async.py:21]: Do we want to mark this as async? `init_firestore_async_client():` and `# [START init_firestore_async_client]`?

**Review by lahirumaramba — APPROVED** [review]
LGTM!

File: snippets/firestore/firestore_async.py (hunk 2 above)
Comment (jkyle109) [reply snippets/firestore/firestore_async.py]: Done

File: snippets/firestore/firestore.py (hunk 1 above)
Comment (jkyle109) [reply snippets/firestore/firestore.py]: Done

**Review by markarndt — APPROVED** [review]

**Pull request merged**